File: Iniciante/URI 1008.c
#include <stdio.h>
 
int main() { 
	int funcionarios, horas;
    float valorPorHora, salario;

    scanf("%d %d %f", &funcionarios, &horas, &valorPorHora);
    salario = horas * valorPorHora;
	printf("NUMBER = %d\n", funcionarios);
    printf("SALARY = U$ %.2f\n", salario);

    return 0;
}

File: Iniciante/URI 1008.go
package main
 
import (
    "fmt"
)
 
func main() {
    var funcionarios int
    var horas, valorPorHora, salario float64

    fmt.Scanf("%d", &funcionarios)
    fmt.Scanf("%f", &horas)
    fmt.Scanf("%f", &valorPorHora)
    salario = valorPorHora * horas
    fmt.Printf("NUMBER = %d\n", funcionarios)
    fmt.Printf("SALARY = U$ %.2f\n", salario)
}